sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -415,19 +415,21 @@ def wrapper(*args, **kwargs):
   return wrapper
 
 
-def not_implemented_method(op, jira='BEAM-9547', base_type=None):
+def not_implemented_method(op, issue='20318', base_type=None):
   """Generate a stub method for ``op`` that simply raises a NotImplementedError.
 
   For internal use only. No backwards compatibility guarantees."""
   assert base_type is not None, "base_type must be specified"
+  issue_url = f"https://issues.apache.org/jira/{issue}." if issue.startswith(
+      "BEAM-") else f"https://github.com/apache/beam/issues/{issue}"
```

Comment:
This is fine for now, to be safe, but let's follow up with a change to remove the else and make sure all the BEAM- references are gone (note to self).

Reply:
That sounds good - I mostly left it because I was only guaranteed to get active issues - ones that have been closed weren't migrated and my tool neither flagged nor changed them.

In theory those should have all been removed when the issue was closed, but my experience would indicate otherwise 😄